web/src/beta/components/DragAndDropList/Item.tsx

```diff
@@ -91,7 +94,16 @@ const Item: FC<Props> = ({
   });
 
   drag(drop(ref));
-  return (
+
+  // eslint-disable-next-line no-prototype-builtins
+  return item.hasOwnProperty("extensionId") ? (
+    <SItem ref={preview} data-handler-id={handlerId} isDragging={isDragging}>
+      <DndHandle ref={ref}>
+        <Icon icon="dndHandle" size={16} />
+      </DndHandle>
+      {children}
+    </SItem>
+  ) : (
```

Comment:
This is:
1) making it so the handle Icon is ALWAYS showing, which isn't good.
2) can only drop successfully if I drag overtop of another item's handle.

Can you try to fix these two issues? 🙏🏻

Reply:
fixed